ISISComputingGroup/BeckhoffPLCCode
# Ticket 3991: Run System Tests

See https://github.com/ISISComputingGroup/IBEX/issues/3991

To test:
* Ensure you have the beckhoff [Twincat XAE](https://www.beckhoff.com/forms/twincat3/warenkorb.aspx?lg=en&title=TC31-Full-Setup.3.1.4022.29&version=3.1.4022.29)
* Ensure you have the [twincat IOC ](https://github.com/ISISComputingGroup/EPICS-ioc/pull/382) and [IOC test framework](https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/pull/184) changes
* Build support/tcIoc, ioc/twincat
* Run the tests as specified in the jenkins file for this repo
* Modify the PLC code in [here](https://github.com/ISISComputingGroup/BeckhoffPLCCode/pull/19/files#diff-8401c245353452d1374fc79a8347bc73) to do something different, confirm tests now fail

Note that this will not pass on Jenkins until the IOC test framework and ioc code is merged
